lms/services/key.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from jose import constants, jwk
+from sqlalchemy import func
+
+from lms.models import Key
+from lms.services import AESService
+
+
+class KeyService:
+    def __init__(self, db, aes_service):
+        self._db = db
+        self._aes = aes_service
```

Comment:
```suggestion
    def __init__(self, db, aes_service: AESService):
        self._db = db
        self._aes_service = aes_service
```

There are quite a few crypto libraries floating about. It's nice to point out this is one of ours. This also matches the naming used elsewhere in services.